src/3rdparty/walkontable/src/overlay/bottom.js

```diff
@@ -207,22 +207,18 @@ class BottomOverlay extends Overlay {
       tableHeight = 0;
     }
 
-    overlayRootStyle.height = `${tableHeight === 0 ? tableHeight : tableHeight}px`;
+    overlayRootStyle.height = `${tableHeight}px`;
   }
 
   /**
    * Adjust overlay root childs size
    */
   adjustRootChildrenSize() {
-    let scrollbarWidth = getScrollbarWidth(this.wot.rootDocument);
+    const { holder } = this.clone.wtTable;
 
     this.clone.wtTable.hider.style.width = this.hider.style.width;
-    this.clone.wtTable.holder.style.width = this.clone.wtTable.holder.parentNode.style.width;
-
-    if (scrollbarWidth === 0) {
-      scrollbarWidth = 30;
-    }
-    this.clone.wtTable.holder.style.height = `${parseInt(this.clone.wtTable.holder.parentNode.style.height, 10) + scrollbarWidth}px`;
+    holder.style.width = holder.parentNode.style.width;
+    holder.style.height = `${parseInt(holder.parentNode.style.height, 10)}px`;
```

Comment:
```suggestion
    holder.style.height = holder.parentNode.style.height;
```
Correct me if I'm wrong but if I understand correctly this line can be simplified by reassigning the height of the parent element directly. Without reparsing from string -> number -> string?

After the changes, the tests still pass.

Reply:
You are right. It's even better. It will be exactly as it should be (eg.: 17.4px). Thank you :+1: